src/Microsoft.ML.Transforms/Microsoft.ML.Transforms.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeInPackage>Microsoft.ML</IncludeInPackage>
     <DefineConstants>CORECLR</DefineConstants>
+    <BaseIntermediateOutputPath>..\..\bin\obj\AnyCPU.Debug\Microsoft.ML.Transforms</BaseIntermediateOutputPath>
```

Comment:
>>..\..\bin\obj\AnyCPU.Debug\Microsoft.ML.Transforms [](start = 31, length = 51)

There seems to be a DEBUG-only path in the general section. This should be replaced with the build type variable - the path is incorrect for the release build.